.github/workflows/pages.yml

```
@@ -3,10 +3,10 @@ name: 'Docusaurus CI/CD'
 on:
   push:
     branches: ['main']
-    paths: ['docs/**', 'blog/**', 'src/**', 'static/**', '**/*.js', 'package.json', 'yarn.lock']
+    paths: ['./docs/**', './blog/**', './src/**', './static/**', './package.json', './yarn.lock']
   pull_request:
```

```suggestion
  pull_request:
    types:
      - opened
      - reopened
      - synchronize
      - closed
```

This will make the CI run even after the PR is closed, to make sure the action is remove the preview environment

reference: https://github.com/rossjrw/pr-preview-action/blob/main/README.md